src/main/java/com/stripe/model/checkout/Session.java

```diff
@@ -323,6 +333,43 @@ public static Session create(SessionCreateParams params, RequestOptions options)
     return ApiResource.request(ApiResource.RequestMethod.POST, url, params, Session.class, options);
   }
 
+  /** Returns a list of Line Items */
+  public LineItemCollection listLineItems(Map<String, Object> params) throws StripeException {
+    return listLineItems(params, (RequestOptions) null);
+  }
+
+  /** Returns a list of Line Items */
+  public LineItemCollection listLineItems(Map<String, Object> params, RequestOptions options)
+      throws StripeException {
+    String url =
+        String.format(
+            "%s%s",
+            Stripe.getApiBase(),
+            String.format(
+                "/v1/checkout/sessions/%s/line_items", ApiResource.urlEncodeId(this.getId())));
+
+    return ApiResource.requestCollection(url, params, LineItemCollection.class, options);
+  }
+
+  /** Returns a list of Line Items */
+  public LineItemCollection listLineItems(SessionLineItemCollectionListParams params)
```

Comment:
I believe the name is backwards. When we confirm a `PaymentIntent` it's `PaymentIntentConfirmParams`. Similarly, this is a nested resource and so usually you do `<operation><nestedresource>` so in ruby it would be `listLineItems` and in the future `createLineItem` `deleteLineItem` etc.

So I believe the correct name for the class should be `SessionListLineItemsParams` instead.